topic/topic.py

```diff
@@ -0,0 +1,19 @@
+from redbot.core import commands
+
+
+class Topic(commands.Cog):
+    """Thus beginith the topic command"""
+
+    def __init__(self):
+        pass
+
+    @commands.command()
+    @commands.guild_only()
+    async def topic(self, ctx: commands.Context):
+        """Repeats the current channel's topic as a message in the channel."""
+
+        topic = ctx.channel.topic
+        if topic:
+            await ctx.send(f"{ctx.channel.mention}: {topic}")
+        else:
+            await ctx.send("This channel does not have a topic.")
```

Comment:
```suggestion
    @app_commands.command(name="topic")
    @app_commands.default_permissions(send_messages=True)
    @app_commands.guild_only()
    async def topic(self, interaction: discord.Interaction):
        """Repeats the current channel's topic as a message in the channel."""

        topic = interaction.channel.topic
        if topic:
            await interaction.response.send_message(f"{interaction.channel.mention}: {topic}")
        else:
            await interaction.response.send_message("This channel does not have a topic.")
```

Reply:
I wonder if we could do it both ways.  I know discord is really pushing these slash commands but I personally don't really like them as the ! commands are faster for me and it doesn't do that popup list I find kinda annoying.
Homelabbers are also creatures of habit, I know many of us prefer the old ways the same way ancient redditors prefer old reddit.  However I do see how slash commands would be easier for the newer folks too.

What's the opinions on implementing it both ways?

Reply:
That's fair enough, I'm certainly happy to keep both.

I definitely understand (and share some of) your issues with slash commands tbh. That said, despite their issues, what I really like about them is it makes feature discovery _so_ much easier, since you don't have to
* Guess/remember/assume a command prefix
* Pour over a help menu to discover commands, for which there is no clear indication of existence to new users, albeit being a fairly obvious command
* Pour over further help menus to figure out how a cog works if it has subcommands, flexible options, etc.

This is all obvious to us, but far from it to people who are new to Homelab or Discord as a whole.

Reply:
I agree.  I have added both and updated the readme to document that it works both ways.